openshift/distro/ubi/9/Dockerfile

```diff
@@ -26,22 +26,19 @@
 #                   with number of cpus being a good heuristic.
 #
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal
+FROM registry.access.redhat.com/ubi9/ubi-minimal
 
 ENV HOME=/home
 WORKDIR "${HOME}/app"
 
-ARG RUBY_VERSION="3.0"
-ARG RUNTIME_DEPS="ruby"
+ARG RUBY_VERSION="3.3"
 
-RUN echo -e "[ruby]\nname=ruby\nstream=${RUBY_VERSION}\nprofiles=\nstate=enabled\n" > /etc/dnf/modules.d/ruby.module \
- && microdnf update --nodocs \
- && microdnf install --nodocs ${RUNTIME_DEPS} \
+RUN microdnf module enable -y "ruby:${RUBY_VERSION}" \
  && chown -R 1001:1001 "${HOME}"
 
 ARG BUILD_DEPS="tar make file findutils git patch gcc automake autoconf libtool redhat-rpm-config openssl-devel ruby-devel"
 
-RUN microdnf install --nodocs ${BUILD_DEPS}
+RUN microdnf install -y --nodocs ${BUILD_DEPS}
```

Comment:
Apparently `microdnf` in RHEL8 didn't stop to query the user. Now we have to add `-y` to ensure the process doesn't require user intervention